packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/Friend.Serialization.cs

```diff
@@ -24,25 +26,127 @@ internal Friend()
         {
         }
 
-        void IJsonModel<Friend>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
+        void IJsonModel<Friend>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options) => WriteCore(writer, options);
```

Comment:
I think we want to add the start/end object in here with WriteCore called in between like this

https://gist.github.com/m-nash/c637584d170566a047942758d34820e7